examples/interface-to-external-code/interface_to_external_code.bal

```diff
@@ -0,0 +1,9 @@
+import ballerina/jballerina.java;
+
+public function open(string path) returns handle|error = external;
+
+// This function invoke register function that accept one string parameter and
+// located inside the com.project.register.Register Java class
+public isolated function register(string name) returns error? = @java:Method {
```

Comment:
Do we need to isolate this function?

Reply:
I guess it will give a warning, If we didn't make it isolated.